source/mesh_system.c

```diff
@@ -61,10 +65,15 @@ void mesh_system_init(void)
         randLIB_seed_random();
         platform_timer_enable();
         eventOS_scheduler_init();
+#ifndef YOTTA_CFG
+        ns_event_loop_thread_create();
+#endif
         trace_init(); // trace system needs to be initialized right after eventOS_scheduler_init
```

Comment:
Don't think it matters (at least at the moment) - that only creates the thread, it doesn't start it. One-line function, without trace.

Guess I was viewing that thread_create call as an extension of the eventOS_scheduler_init().